collection_north.gemspec

```diff
@@ -7,15 +7,22 @@ require 'collection_north/version'
 Gem::Specification.new do |s|
   s.name        = 'collection_north'
   s.version     = CollectionNorth::VERSION
-  s.authors     = ['criedlberger']
-  s.email       = ['riedlber@ualberta.ca']
+  s.authors     = ['Matt Barnett', 'Shane Murnaghan']
+  s.email       = ['mbarnett@ualberta.ca', 'murnagha@ualberta.ca']
```

Comment:
I was going to add everyone else on the team here but not sure if they want their emails public haha.